code/datums/diseases/critical.dm

```diff
@@ -0,0 +1,115 @@
+/datum/disease/shock
```

Comment:
Could use buff/debuff framework for this? I mean we do have it.

Reply:
the system works better as a disease, as it will show up on medical HUDs. Also just the natural of progression/removal and having it died to reagents and based on stages? It's a perfect shoe-in---I'd have to re-create effectively the same systems for a status effect.